### DISCUSSION TITLE
How to obtain sequence (after basecall) that matches second chunk?

### DISCUSSION THREAD

**Discussion by jennieli421** [LooseLab/readfish#331] [Q&A]
Suppose that I allow for maximum 3 chunks. First chunk failed. At next decision, by default, the first and second chunk are concatenated. How to only get the sequences matching only chunk two? Similarly, if proceed to third chunk, how to extract sequences matching only chunk three?

**Comment by jennieli421** [comment]
@Adoni5 @mattloose I am following up on this question in case people didn't see.

**Comment by mattloose** [comment]
HI,

We don't support this as an option in readfish. In principle you could work out the additional sequence that has been generated by looking at the log files that readfish writes out when running - by comparing one to another you would be able to infer the new chunk. Altenratively you could look at modifying the code such that only the new received chunk is called. 

I'm not sure what the use case for this would be though and its not something we would look to implement at this time.

Matt
